presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix rare (but practical) incorrect query result for queries involving timestamp,
+  when the time zone database embedded in the version of Joda-Time that Presto depends on
+  does not match the time zone database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when optimize_mixed_distinct_aggregation is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add :func:`reduce_agg` aggregate function.
+* Add :func:`milliseconds` function (issue #9524).
+* Add support for predicate pushdown for partition symbols for window operators.
+* Remove SHOW STATS ON syntax
+* Restrict WHERE clause in SHOW STATS to filters that can be pushed down
+* Remove ``node_id`` column from ``system.runtime.queries`` table.
+* Immediately return final results to client for a failed query.
+
+Web UI
+---------------
+
+* Fix failure when rendering live plan view for queries involving index joins (issue #12054).
+
+Hive Changes
+---------------
+
+* Queries reading Parquet files will now fail by default if statistics
+  in those Parquet files are corrupt (e.g., min > max).
+  To disable this behavior, set configuration property ``hive.parquet.fail-on-corrupted-statistics``
+  or session property ``parquet_fail_with_corrupted_statistics`` to false.
+
+Kudu Changes
+---------------
+
+* Update Kudu client to 1.8.0.
+* Add Kudu table property ``number_of_replicas`` to ``SHOW CREATE TABLE`` output.
+
+Cassandra Changes
+---------------
+
+* Add configuration properties ``cassandra.splits-per-node`` and ``cassandra.protocol-version``
+ to allow connecting to Cassandra services older than 2.1.5.
+
+MySQL, PostgreSQL, Redshift, and SQL Server Changes
+-------------
+
+* Add support for predicate pushdown for ``char(x)`` columns.
+
+Verifier
+---------------
+
+* Add config run-teardown-on-result-mismatch. When set to false,
```

Comment:
surround "run-teardown-on-result-mismatch" with double backticks like the other configuration properties

Reply:
```
Add ``run-teardown-on-result-mismatch`` config property. ...
```